flink-container/src/main/java/org/apache/flink/container/entrypoint/StandaloneJobClusterConfigurationParserFactory.java

```diff
@@ -92,9 +105,23 @@ public StandaloneJobClusterConfiguration createResult(@Nonnull CommandLine comma
 			restPort,
 			savepointRestoreSettings,
 			jobId,
+			jobIdSeed,
 			jobClassName);
 	}
 
+	private String getJobIdSeed(@Nonnull final CommandLine commandLine) throws FlinkParseException {
+
+		boolean isJobIdSeedConfigured = commandLine.hasOption(JOB_ID_SEED_OPTION.getOpt());
+		boolean isJobIdConfigured = commandLine.hasOption(JOB_ID_OPTION.getOpt());
+
+		if (isJobIdSeedConfigured && isJobIdConfigured) {
```

Comment:
You are right. I will update the ticket description. The idea is the following: 

* It never makes sense to configure both, hence fail early in the ParserFactory. 
* The `StandaloneJobClusterEntrypoint` does not make any assumptions about the exclusiveness enforced by the ParserFactory and resolves the `JobID` as described.

Reply:
It is more a design decision if they should be mutually exclusive or not. I just pointed out that the old ticket description was mismatching the implementation.

If you want to keep them mutually exclusive, what about removing the added `--job-id-seed` in the `job-cluster-job.yaml.template` as command line argument? It's possible, that this file is used by others as Kubernetes resource template. Now the template explicitly uses the `--job-id-seed` and it's not compatible with old `--job-id` argument in that case.

Reply:
I would rather change the README so that you specify the `--job-id` explicitly, you have to remove the `--job-id-seed` parameter. 

The templates are an opinionated starting point to run Job/Application Clusters on Kubernetes, as such they should contain the `--job-id-seed`.

What do you think?

Reply:
I added a warning to the README for the time being.